packages/pluginsdk/package.json

```diff
@@ -13,37 +13,39 @@
     "prepublishOnly": "npm run build"
   },
   "bin": {
-    "scaffold": "scripts/scaffold.js",
     "check-plugin": "scripts/check-plugin.js",
-    "dev-proxy": "scripts/dev-proxy/server.js"
+    "dev-proxy": "scripts/dev-proxy/server.js",
+    "restore-scaffold-file": "scripts/restore-scaffold-file.js",
+    "scaffold": "scripts/scaffold.js",
+    "write-json": "scripts/write-json.js"
   },
   "dependencies": {
-    "check-peer-dependencies": "^2.0.1",
-    "chokidar": "^3.4.0",
+    "check-peer-dependencies": "^2.0.3",
+    "chokidar": "^3.4.2",
     "express": "^4.17.1",
-    "http-proxy-middleware": "^1.0.4",
-    "lodash": "^4.17.15",
+    "http-proxy-middleware": "^1.0.5",
+    "lodash": "^4.17.20",
     "readline-sync": "^1.4.10",
-    "strip-json-comments": "^3.1.0",
-    "ws": "^7.3.0",
-    "yargs": "^15.3.1"
+    "strip-json-comments": "^3.1.1",
+    "ws": "^7.3.1",
+    "yargs": "^16.0.3"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "11.0.2",
-    "@rollup/plugin-node-resolve": "7.1.3",
-    "@rollup/plugin-typescript": "4.1.1",
-    "@spinnaker/core": "0.0.479",
-    "rollup": "1.32.1",
-    "rollup-plugin-external-globals": "0.5.0",
+    "@rollup/plugin-commonjs": "15.0.0",
+    "@rollup/plugin-node-resolve": "9.0.0",
+    "@rollup/plugin-typescript": "6.0.0",
+    "@spinnaker/core": "0.0.506",
+    "rollup": "2.26.11",
+    "rollup-plugin-external-globals": "0.6.0",
     "rollup-plugin-less": "1.1.2",
-    "rollup-plugin-postcss": "3.1.1",
-    "typescript": "3.8.3"
+    "rollup-plugin-postcss": "3.1.8",
+    "typescript": "4.0.2"
   },
   "peerDependencies": {
-    "@rollup/plugin-commonjs": "11.0.2",
-    "@rollup/plugin-node-resolve": "7.1.3",
-    "@rollup/plugin-typescript": "4.1.1",
-    "@spinnaker/core": "0.0.479",
+    "@rollup/plugin-commonjs": "15.0.0",
+    "@rollup/plugin-node-resolve": "9.0.0",
+    "@rollup/plugin-typescript": "6.0.0",
+    "@spinnaker/core": "0.0.506",
```

Comment:
@christopherthielen Do you mind upgrading this to the latest (`0.0.507`) while you have the PR open?